snap/snapcraft.yaml

```diff
@@ -162,15 +150,15 @@ parts:
       echo "Starting at $(date)"
       START_TIME=$(date +%s)
 
-      CPU_COUNT=$(lscpu --parse=CPU | grep -v "^#" | wc -l)
-
-      export DESTDIR="$(readlink --canonicalize ${SNAPCRAFT_PART_INSTALL})"
+      DESTDIR="${SNAPCRAFT_PART_INSTALL}"
 
       ./otp_build autoconf
       ./configure --prefix=/usr
 
-      make -j ${CPU_COUNT}
-      make install
+      make -j ${SNAPCRAFT_PARALLEL_BUILD_COUNT}
+      echo "[DEBUG] done with building"
```

Comment:
@cmatsuoka If you look at the CI output then this line is never printed. It is as if the `build` step is not completing.